lib/client.js

```diff
@@ -273,6 +274,9 @@ Client.prototype.close = function (done) {
     conn.removeAllListeners('error')
     conn.on('error', nop)
 
+    that.connected = false
+    that.disconnected = true
```

Comment:
is disconnected a new property?

Reply:
`disconnected` is a existing property.
It is defined in https://github.com/mcollina/aedes/blob/master/lib/client.js#L46
and also be used in https://github.com/mcollina/aedes/blob/master/lib/handlers/index.js#L54